client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -211,7 +211,32 @@ export class HubConnection {
     }
 
     on(methodName: string, method: (...args: any[]) => void) {
-        this.methods.set(methodName.toLowerCase(), method);
+        if (!methodName || !method) {
+            return;
+        }
+
+        methodName = methodName.toLowerCase();
+        if (!this.methods.has(methodName)) {
+            this.methods.set(methodName, []);
+        }
+
+        this.methods.get(methodName).push(method);
+    }
+
+    off(methodName: string, method: (...args: any[]) => void) {
+        if (!methodName || !method) {
+            return;
+        }
+
+        methodName = methodName.toLowerCase();
+        let handlers = this.methods.get(methodName);
+        if (!handlers) {
+            return;
+        }
+        var removeIdx = handlers.indexOf(method);
```

Comment:
This works?

Reply:
Yes - as long as this is the same function. I also added a test.

Reply:
Heh cool. I guess it's just object instance.